example_configs/gogs.md

```diff
@@ -0,0 +1,44 @@
+# Gogs LDAP configuration
+
+Gogs can make use of LDAP and therefore lldap.
+
+The following configuration is adapted from the example configuration at [their repository](https://github.com/thchha/lldap/new/main/example_configs).
+The example is a container configuration - the file should just needs to live within `conf/auth.d/some_name.conf`:
+
+```
```

Comment:
No effect so omitted.

Reply:
Wouldn't it be toml? If you remove the `cat` line